tests/integration_tests/datasources/test_azure.py

```diff
@@ -45,3 +48,43 @@ def test_azure_eject(session_cloud: IntegrationCloud):
                 session_cloud.cloud_instance.delete_image(snapshot_id)
         else:
             _check_for_eject_errors(instance)
+
+
+@pytest.mark.skipif(PLATFORM != "azure", reason="Test is Azure specific")
+def test_azure_multi_nic_setup(session_cloud: IntegrationCloud) -> None:
+    """Integration test for https://warthogs.atlassian.net/browse/CPC-3999
+
+    Azure should have the primary NIC only route to DNS.
+    Ensure other NICs do not have route to DNS.
```

Comment:
I do not believe this test exercises the behaviour of concern. As described in https://warthogs.atlassian.net/browse/CPC-3999 , there is no route to the DHCP/DNS server through the secondary NIC on Mantic hosts (per `ip route`), but the DNS resolver has still associated the DNS server with the secondary NIC and thus transmits DNS queries through that NIC. As such, I believe that this test should instead verify that there is no longer a DNS server associated with the secondary NIC in the output of `resolvectl dns`.

Reply:
I agree. Let me see how parse `resolvectl dns`. Thanks for this.

Reply:
Bionic does not have `resolvectl`, so that will be interesting....will figure out how to update the test.